integration/docker/Dockerfile-dev

```diff
@@ -9,86 +9,24 @@
 # See the NOTICE file distributed with this work for information regarding copyright ownership.
 #
 
-# ARG defined before the first FROM can be used in FROM lines
-ARG JAVA_VERSION=8
+FROM alluxio/alluxio:2.9.0-SNAPSHOT
```

Comment:
we may need to update the release related version change scripts? so that during release this version can change to target version as well

Reply:
Thanks for reminding. No change is needed here.